Makefile

```diff
@@ -45,6 +45,12 @@ linux-prepare::
 	if [ "x$$BINFMT_WARN" = "xyes" ]; then \
 		cat Documentation/binfmt_misc-warning-Linux.txt ; \
 	fi; \
+	if  [ -f build-tools/scripts/dependencies/linux-prepare-$(LINUX_DISTRO).sh ] || [ -f build-tools/scripts/dependencies/linux-prepare-$(LINUX_DISTRO)-$(LINUX_DISTRO_RELEASE).sh ]  &&  [ "$(NO_SUDO)" = "false" ]; then \
```

Comment:
well, it's to prevent copy-pasting that message and it's to clear up that the script wants to install packages.